www-apps/gitea/gitea-1.9.3.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-vcs-snapshot tmpfiles systemd
+
+EGO_PN="code.gitea.io/gitea"
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
```

Comment:
This is definitely insufficient. The vendored libs definitely include BSD, Apache-2.0 and MPL-2.0 licenses, all more restrictive than MIT. Furthermore, I don't recognize the license in `./github.com/couchbase/goutils/LICENSE.md`.

You need to establish if the licenses are compatible, and establish the effective license.

Reply:
Actually, that couchbase license looks like an EULA to me. @gentoo/licenses ?

I don't want to give you advices on how to proceed here before I get a reply from someone more competent than myself. I'll file a bug for this.

Reply:
any thing new here ? seems like other distro's also package it with just refering to "MIT" https://repology.org/project/gitea/versions ...
I could add `BSD Apache-2.0 MPL-2.0` to `LICENSE`
and `EULA` and install the couchbase license-file.

Reply:
Have you actually checked the licenses yourself, or merely copied what I said?

Reply:
I had a brief look (about 164 modules - I'm not a license expert)

A closer look is gives my: Apache-2.0 BSD-2 BSD ISC MPL-2.0 and ??
???            [github.com/couchbase/goutils/LICENSE.md](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/couchbase/goutils/LICENSE.md)
??? MIT-statement missing?           [github.com/glycerine/go-unsnap-stream/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/glycerine/go-unsnap-stream/LICENSE)
???icu-license [github.com/gogits/chardet/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gogits/chardet/LICENSE)
???+MIT        [gopkg.in/yaml.v2/LICENSE.libyaml](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/yaml.v2/LICENSE.libyaml)

APACHE-2.0     [github.com/blevesearch/bleve/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/blevesearch/bleve/LICENSE)
APACHE-2.0     [github.com/blevesearch/segment/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/blevesearch/segment/LICENSE)
APACHE-2.0     [github.com/bradfitz/gomemcache/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/bradfitz/gomemcache/LICENSE)
APACHE-2.0     [github.com/couchbase/vellum/levenshtein2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/couchbase/vellum/levenshtein2/LICENSE)
APACHE-2.0     [github.com/couchbase/vellum/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/couchbase/vellum/LICENSE)
APACHE-2.0     [github.com/go-macaron/binding/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/binding/LICENSE)
APACHE-2.0     [github.com/go-macaron/cache/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/cache/LICENSE)
APACHE-2.0     [github.com/go-macaron/captcha/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/captcha/LICENSE)
APACHE-2.0     [github.com/go-macaron/cors/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/cors/LICENSE)
APACHE-2.0     [github.com/go-macaron/csrf/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/csrf/LICENSE)
APACHE-2.0     [github.com/go-macaron/i18n/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/i18n/LICENSE)
APACHE-2.0     [github.com/go-macaron/inject/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/inject/LICENSE)
APACHE-2.0     [github.com/go-macaron/session/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/session/LICENSE)
APACHE-2.0     [github.com/go-macaron/toolbox/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-macaron/toolbox/LICENSE)
APACHE-2.0     [github.com/go-openapi/analysis/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/analysis/LICENSE)
APACHE-2.0     [github.com/go-openapi/errors/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/errors/LICENSE)
APACHE-2.0     [github.com/go-openapi/jsonpointer/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/jsonpointer/LICENSE)
APACHE-2.0     [github.com/go-openapi/jsonreference/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/jsonreference/LICENSE)
APACHE-2.0     [github.com/go-openapi/loads/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/loads/LICENSE)
APACHE-2.0     [github.com/go-openapi/runtime/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/runtime/LICENSE)
APACHE-2.0     [github.com/go-openapi/spec/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/spec/LICENSE)
APACHE-2.0     [github.com/go-openapi/spec/license.go](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/spec/license.go)
APACHE-2.0     [github.com/go-openapi/strfmt/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/strfmt/LICENSE)
APACHE-2.0     [github.com/go-openapi/swag/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/swag/LICENSE)
APACHE-2.0     [github.com/go-openapi/validate/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/validate/LICENSE)
APACHE-2.0     [github.com/go-swagger/go-swagger/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-swagger/go-swagger/LICENSE)
APACHE-2.0     [github.com/matttproud/golang_protobuf_extensions/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/matttproud/golang_protobuf_extensions/LICENSE)
APACHE-2.0     [github.com/mschoch/smat/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mschoch/smat/LICENSE)
APACHE-2.0     [github.com/pquerna/otp/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/pquerna/otp/LICENSE)
APACHE-2.0     [github.com/prometheus/client_golang/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/prometheus/client_golang/LICENSE)
APACHE-2.0     [github.com/prometheus/client_model/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/prometheus/client_model/LICENSE)
APACHE-2.0     [github.com/prometheus/common/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/prometheus/common/LICENSE)
APACHE-2.0     [github.com/prometheus/procfs/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/prometheus/procfs/LICENSE)
APACHE-2.0     [github.com/RoaringBitmap/roaring/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/RoaringBitmap/roaring/LICENSE)
APACHE-2.0     [github.com/RoaringBitmap/roaring/LICENSE-2.0.txt](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/RoaringBitmap/roaring/LICENSE-2.0.txt)
APACHE-2.0     [github.com/spf13/afero/LICENSE.txt](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/spf13/afero/LICENSE.txt)
APACHE-2.0     [github.com/Unknwon/cae/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/Unknwon/cae/LICENSE)
APACHE-2.0     [github.com/Unknwon/com/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/Unknwon/com/LICENSE)
APACHE-2.0     [github.com/Unknwon/i18n/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/Unknwon/i18n/LICENSE)
APACHE-2.0     [github.com/Unknwon/paginater/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/Unknwon/paginater/LICENSE)
APACHE-2.0     [github.com/xanzy/ssh-agent/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/xanzy/ssh-agent/LICENSE)
APACHE-2.0     [github.com/yohcop/openid-go/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/yohcop/openid-go/LICENSE)
APACHE-2.0     [go.mongodb.org/mongo-driver/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/go.mongodb.org/mongo-driver/LICENSE)
APACHE-2.0     [google.golang.org/appengine/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/google.golang.org/appengine/LICENSE)
APACHE-2.0     [gopkg.in/ini.v1/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/ini.v1/LICENSE)
APACHE-2.0     [gopkg.in/macaron.v1/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/macaron.v1/LICENSE)
APACHE-2.0     [gopkg.in/src-d/go-billy.v4/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/src-d/go-billy.v4/LICENSE)
APACHE-2.0     [gopkg.in/src-d/go-git.v4/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/src-d/go-git.v4/LICENSE)
APACHE-2.0     [gopkg.in/yaml.v2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/yaml.v2/LICENSE)
BSD-2          [github.com/andybalholm/cascadia/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/andybalholm/cascadia/LICENSE)
BSD-2          [github.com/denisenkom/go-mssqldb/LICENSE.txt](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/denisenkom/go-mssqldb/LICENSE.txt)
BSD-2          [github.com/golang/snappy/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/golang/snappy/LICENSE)
BSD-2          [github.com/google/go-github/v24/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/google/go-github/v24/LICENSE)
BSD-2          [github.com/google/go-querystring/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/google/go-querystring/LICENSE)
BSD-2          [github.com/go-redis/redis/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-redis/redis/LICENSE)
BSD-2          [github.com/gorilla/handlers/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gorilla/handlers/LICENSE)
BSD-2          [github.com/keybase/go-crypto/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/keybase/go-crypto/LICENSE)
BSD-2          [github.com/klauspost/compress/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/klauspost/compress/LICENSE)
BSD-2          [github.com/magiconair/properties/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/magiconair/properties/LICENSE)
BSD-2          [github.com/msteinert/pam/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/msteinert/pam/LICENSE)
BSD-2          [github.com/pkg/errors/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/pkg/errors/LICENSE)
BSD-2          [github.com/PuerkitoBio/urlesc/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/PuerkitoBio/urlesc/LICENSE)
BSD-2          [github.com/russross/blackfriday/LICENSE.txt](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/russross/blackfriday/LICENSE.txt)
BSD-2          [github.com/siddontang/go-snappy/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/siddontang/go-snappy/LICENSE)
BSD-2          [github.com/willf/bitset/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/willf/bitset/LICENSE)
BSD-2          [golang.org/x/crypto/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/golang.org/x/crypto/LICENSE)
BSD-2          [golang.org/x/net/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/golang.org/x/net/LICENSE)
BSD-2          [golang.org/x/oauth2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/golang.org/x/oauth2/LICENSE)
BSD-2          [golang.org/x/sys/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/golang.org/x/sys/LICENSE)
BSD-2          [golang.org/x/text/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/golang.org/x/text/LICENSE)
BSD-2          [golang.org/x/tools/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/golang.org/x/tools/LICENSE)
BSD-2          [gopkg.in/asn1-ber.v1/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/asn1-ber.v1/LICENSE)
BSD-2          [gopkg.in/bufio.v1/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/bufio.v1/LICENSE)
BSD-2          [gopkg.in/redis.v2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/redis.v2/LICENSE)
BSD-2          [gopkg.in/warnings.v0/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/warnings.v0/LICENSE)
BSD-2+ISC      [github.com/emirpasic/gods/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/emirpasic/gods/LICENSE)
BSD            [github.com/edsrzf/mmap-go/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/edsrzf/mmap-go/LICENSE)
BSD            [github.com/facebookgo/stats/license](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/facebookgo/stats/license)
BSD            [github.com/fsnotify/fsnotify/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/fsnotify/fsnotify/LICENSE)
BSD            [github.com/golang/protobuf/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/golang/protobuf/LICENSE)
BSD            [github.com/gorilla/context/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gorilla/context/LICENSE)
BSD            [github.com/gorilla/mux/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gorilla/mux/LICENSE)
BSD            [github.com/gorilla/securecookie/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gorilla/securecookie/LICENSE)
BSD            [github.com/gorilla/sessions/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gorilla/sessions/LICENSE)
BSD            [github.com/go-xorm/xorm/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-xorm/xorm/LICENSE)
BSD            [github.com/jessevdk/go-flags/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/jessevdk/go-flags/LICENSE)
BSD            [github.com/klauspost/crc32/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/klauspost/crc32/LICENSE)
BSD            [github.com/lunny/log/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/lunny/log/LICENSE)
BSD            [github.com/pmezard/go-difflib/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/pmezard/go-difflib/LICENSE)
BSD            [github.com/PuerkitoBio/goquery/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/PuerkitoBio/goquery/LICENSE)
BSD            [github.com/PuerkitoBio/purell/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/PuerkitoBio/purell/LICENSE)
BSD            [github.com/spf13/pflag/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/spf13/pflag/LICENSE)
BSD            [github.com/src-d/gcfg/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/src-d/gcfg/LICENSE)
BSD            [github.com/syndtr/goleveldb/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/syndtr/goleveldb/LICENSE)
BSD            [mvdan.cc/xurls/v2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/mvdan.cc/xurls/v2/LICENSE)
BSD            [xorm.io/builder/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/xorm.io/builder/LICENSE)
BSD            [xorm.io/core/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/xorm.io/core/LICENSE)
ISC            [github.com/davecgh/go-spew/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/davecgh/go-spew/LICENSE)
ISC            [github.com/nfnt/resize/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/nfnt/resize/LICENSE)
MIT            [github.com/asaskevich/govalidator/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/asaskevich/govalidator/LICENSE)
MIT            [github.com/beorn7/perks/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/beorn7/perks/LICENSE)
MIT            [github.com/blevesearch/go-porterstemmer/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/blevesearch/go-porterstemmer/LICENSE)
MIT            [github.com/boombuler/barcode/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/boombuler/barcode/LICENSE)
MIT            [github.com/chaseadamsio/goorgeous/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/chaseadamsio/goorgeous/LICENSE)
MIT            [github.com/couchbase/gomemcached/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/couchbase/gomemcached/LICENSE)
MIT            [github.com/couchbaselabs/go-couchbase/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/couchbaselabs/go-couchbase/LICENSE)
MIT            [github.com/dgrijalva/jwt-go/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/dgrijalva/jwt-go/LICENSE)
MIT            [github.com/etcd-io/bbolt/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/etcd-io/bbolt/LICENSE)
MIT            [github.com/ethantkoenig/rupture/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/ethantkoenig/rupture/LICENSE)
MIT            [github.com/facebookgo/clock/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/facebookgo/clock/LICENSE)
MIT            [github.com/gogits/cron/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/gogits/cron/LICENSE)
MIT            [github.com/go-openapi/runtime/middleware/denco/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-openapi/runtime/middleware/denco/LICENSE)
MIT            [github.com/go-stack/stack/LICENSE.md](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-stack/stack/LICENSE.md)
MIT            [github.com/issue9/identicon/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/issue9/identicon/LICENSE)
MIT            [github.com/jaytaylor/html2text/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/jaytaylor/html2text/LICENSE)
MIT            [github.com/jbenet/go-context/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/jbenet/go-context/LICENSE)
MIT            [github.com/kballard/go-shellquote/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/kballard/go-shellquote/LICENSE)
MIT            [github.com/kevinburke/ssh_config/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/kevinburke/ssh_config/LICENSE)
MIT            [github.com/klauspost/cpuid/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/klauspost/cpuid/LICENSE)
MIT            [github.com/kr/pretty/License](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/kr/pretty/License)
MIT            [github.com/kr/text/License](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/kr/text/License)
MIT            [github.com/lafriks/xormstore/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/lafriks/xormstore/LICENSE)
MIT            [github.com/lib/pq/LICENSE.md](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/lib/pq/LICENSE.md)
MIT            [github.com/lunny/dingtalk_webhook/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/lunny/dingtalk_webhook/LICENSE)
MIT            [github.com/lunny/levelqueue/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/lunny/levelqueue/LICENSE)
MIT            [github.com/lunny/nodb/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/lunny/nodb/LICENSE)
MIT            [github.com/mailru/easyjson/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mailru/easyjson/LICENSE)
MIT            [github.com/markbates/goth/LICENSE.txt](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/markbates/goth/LICENSE.txt)
MIT            [github.com/mattn/go-isatty/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mattn/go-isatty/LICENSE)
MIT            [github.com/mattn/go-sqlite3/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mattn/go-sqlite3/LICENSE)
MIT            [github.com/mcuadros/go-version/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mcuadros/go-version/LICENSE)
MIT            [github.com/mitchellh/go-homedir/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mitchellh/go-homedir/LICENSE)
MIT            [github.com/mitchellh/mapstructure/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/mitchellh/mapstructure/LICENSE)
MIT            [github.com/oliamb/cutter/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/oliamb/cutter/LICENSE)
MIT            [github.com/pelletier/go-toml/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/pelletier/go-toml/LICENSE)
MIT            [github.com/philhofer/fwd/LICENSE.md](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/philhofer/fwd/LICENSE.md)
MIT            [github.com/satori/go.uuid/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/satori/go.uuid/LICENSE)
MIT            [github.com/sergi/go-diff/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/sergi/go-diff/LICENSE)
MIT            [github.com/shurcooL/httpfs/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/shurcooL/httpfs/LICENSE)
MIT            [github.com/shurcooL/sanitized_anchor_name/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/shurcooL/sanitized_anchor_name/LICENSE)
MIT            [github.com/shurcooL/vfsgen/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/shurcooL/vfsgen/LICENSE)
MIT            [github.com/spf13/cast/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/spf13/cast/LICENSE)
MIT            [github.com/spf13/jwalterweatherman/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/spf13/jwalterweatherman/LICENSE)
MIT            [github.com/spf13/viper/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/spf13/viper/LICENSE)
MIT            [github.com/steveyen/gtreap/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/steveyen/gtreap/LICENSE)
MIT            [github.com/stretchr/testify/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/stretchr/testify/LICENSE)
MIT            [github.com/tinylib/msgp/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/tinylib/msgp/LICENSE)
MIT            [github.com/toqueteos/webbrowser/LICENSE.md](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/toqueteos/webbrowser/LICENSE.md)
MIT            [github.com/tstranex/u2f/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/tstranex/u2f/LICENSE)
MIT            [github.com/urfave/cli/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/urfave/cli/LICENSE)
MIT            [gopkg.in/alexcesaro/quotedprintable.v3/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/alexcesaro/quotedprintable.v3/LICENSE)
MIT            [gopkg.in/editorconfig/editorconfig-core-go.v1/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/editorconfig/editorconfig-core-go.v1/LICENSE)
MIT            [gopkg.in/gomail.v2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/gomail.v2/LICENSE)
MIT            [gopkg.in/ldap.v3/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/ldap.v3/LICENSE)
MIT            [gopkg.in/testfixtures.v2/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/gopkg.in/testfixtures.v2/LICENSE)
MIT            [strk.kbt.io/projects/go/libravatar/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/strk.kbt.io/projects/go/libravatar/LICENSE)
MPL-2.0        [github.com/go-sql-driver/mysql/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/go-sql-driver/mysql/LICENSE)
MPL-2.0        [github.com/hashicorp/hcl/LICENSE](https://github.com/go-gitea/gitea/blob/master/vendor/github.com/hashicorp/hcl/LICENSE)